### PR TITLE
Add profile for running DMN TCK tests during our build

### PR DESCRIPTION
**Pull request by baldimir**
@etirelli @tarilabs can you please review?

### CODE REVIEW

File: kie-dmn/kie-dmn-tck/pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <version>1.9.5</version>
+            <executions>
+              <execution>
+                <id>run-tck-suite</id>
+                <phase>install</phase>
+                <goals>
+                  <goal>bootstrap</goal>
+                </goals>
+                <configuration>
+                  <connectionUrl>scm:git:https://github.com/agilepro/dmn-tck.git</connectionUrl>
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-tck/pom.xml:56]: +1

**Review by etirelli — APPROVED** [review]

**Comment by etirelli** [comment]
@baldimir not sure why is this failing?

**Comment by baldimir** [comment]
jenkins retest this (fail not related to this)

**Comment by baldimir** [comment]
Minor change - drools.version property must be passed with install goal, because systemProperties are passed just to JUnit tests and not the whole build.

**Comment by psiroky** [comment]
One thing that just crossed my mind. Do we want to always clone the HEAD of a master branch? This may very well make the build non-deterministic and possibly introduce some other issues with build reproducibility.

Just something to check. I know the TCK would only be executed on demand via a custom profile, so it may very well not be a big of an issue.

**Comment by baldimir** [comment]
For now yes. We want to test with the latest tests from TCK, because AFAIK it is still in development.

**Comment by etirelli** [comment]
correct, tck is in development and only has master branch. We want the integrated build on a profile to make it easier to detect brakes, but not as part of the continuos build for everyone.

**Pull request merged**